### ISSUE TITLE
Pods being put into multiple folders

### ISSUE THREAD

**Issue by elchefe** [CocoaPods/CocoaPods#4775]
We recently we upgraded CocoaPods to 0.39 and ran a ‘pod install’

Our iOS Jenkins builds are failing with this message:

	fatal error: too many errors emitted, stopping now 

Here are a couple example errors from the console output:

```In file included from Pods/AFNetworking/AFNetworking/AFHTTPClient.h:24:
Pods/AFNetworking/AFNetworking/AFURLConnectionOperation.h:105:38: error: property has a previous declaration
@property (nonatomic, strong) NSSet *runLoopModes;```


```In file included from /Users/jenkins/.jenkins/jobs/A_JENKINS_JOB/workspace/jenkins_ios_target/AN_XCODE_TARGET/build/AFNetworking.h:29:
/Users/jenkins/.jenkins/jobs/A_JENKINS_JOB/workspace/jenkins_ios_target/AN_XCODE_TARGET/build/AFURLConnectionOperation.h:105:38: note: property declared here
@property (nonatomic, strong) NSSet *runLoopModes;```


Thus it seems that AFNetworking is put into 2 different folders ("build" and "pods") and the compiler is erroring out due to duplicate declarations.

the 2 different folders AFNetworking is being put into:
/Users/jenkins/.jenkins/jobs/A_JENKINS_JOB/workspace/jenkins_ios_target/AN_XCODE_TARGET/build
/Users/jenkins/.jenkins/jobs/A_JENKINS_JOB/workspace/jenkins_ios_target/AN_XCODE_TARGET/pods

Looking at the Xcode Build Phases section I see a “Copy Pods Resources” step but no script specified and an “Embed Pods Frameworks” step specifying this:

"${SRCROOT}/Pods/Target Support Files/Pods/Pods-frameworks.sh"


Can anyone please tell me why AFNetworking is put into the build folder AND the pods folder now? Previously I believe it was only put into the pods folder.

thank you for any help

**Comment by segiddins** [comment]
Can you share a project that demonstrates the issue?

**Comment by elchefe** [comment]
Since this is a proprietary application I attempted to strip it down to a form that's shareable but that wasn't feasible. What else can I provide you? Individual files? Brief conf call with screen share? 
Thank you for any help.

**Comment by elchefe** [comment]
Here are screen prints of the differences I see in the build folder:

/Users/jenkins/.jenkins/jobs/A_JENKINS_JOB/workspace/jenkins_ios_target/AN_XCODE_TARGET/build

Before:

![cocoapods_issue_4775_build_folder_contents_before](https://cloud.githubusercontent.com/assets/16726062/12400334/fbddcbd0-bde4-11e5-8c23-d0d1590fc7c9.png)

After:

![cocoapods_issue_4775_build_folder_contents_after](https://cloud.githubusercontent.com/assets/16726062/12400340/03510724-bde5-11e5-9d47-72c4f00ace41.png)

Thank you.

**Comment by manuyavuz** [comment]
Can you please share your podfile?

**Comment by elchefe** [comment]
Sure thing. Here you go: 
[Podfile.txt](https://github.com/CocoaPods/CocoaPods/files/94958/Podfile.txt)

Thank you for looking into this.

**Comment by manuyavuz** [comment]
@elchefe what were you using before 0.39.0? Can you try deintegrating the project and running pod install again, and check if problem continues? Here are the sample steps:

```
gem install cocoapods-deintegrate
pod deintegrate
pod install
```

**Comment by elchefe** [comment]
@manuyavuz The prior version was 0.36.3. I can try the deintegration. Thanks.

**Comment by elchefe** [comment]
@manuyavuz I performed those steps but the issue still remains. Do you know if CP changed anything that would put the header files into the 'build' folder?

**Comment by elchefe** [comment]
@manuyavuz  @segiddins Hi guys, just wanted to let you know that we downgraded cocoapods to 0.38.2 and ran "pod install". After this our Jenkins jobs succeeded without the "fatal error: too many errors emitted, stopping now" issue. The .h files are no longer being put into the 'build' folder and causing those "previous declaration" errors. Though our issue is now resolved it would be helpful to know that if we attempt to upgrade CocoaPods again in the future it wont break our build process. Thank you.
